README.md

```diff
@@ -2,23 +2,23 @@
 
 ## What is Olympus?
 
-Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset-backing and protocol owned value.
+Olympus is an algorithmic currency protocol based on the $OHM token. It introduces unique economic and game-theoretic dynamics into the market through asset-backing and protocol-owned value. Read more about the game theory of Olympus [here](https://olympusdao.medium.com/the-game-theory-of-olympus-e4c5f19a77df).
 
 ## What is the point of Olympus?
 
 Our goal is to build a policy-controlled currency system, in which the behavior of the $OHM token is controlled at a high level by the DAO. In the long term, we believe this system can be used to optimize for stability and consistency so that $OHM can function as a global unit-of-account and medium-of-exchange currency. In the short term, we intend to optimize the system for growth and wealth creation.
 
 ## How do I participate in Olympus?
 
-There are two main strategies for market participants: staking and bonding. Stakers lock supply and receive a share of protocol profits as they are earned. Bonders provide locked liquidity and receive a fixed profit after a fixed period of time.
+There are two main strategies for market participants: staking and bonding. Stakers lock their $OHM tokens in return for the protocol profit, while bonders provide locked liquidity in exchange for $OHM tokens after a fixed period of time.
```

Comment:
- Maybe we could try preventing the term "locking" since we do not have any lockups implemented. Stakers just stake voluntarily. 
- Trying to get DAI bonds into the mix. Your call if this is cool. 

```suggestion
There are two main strategies for market participants: staking and bonding. Stakers stake their $OHM tokens in return for more $OHM tokens, while bonders provide LP or $DAI tokens in exchange for discounted $OHM tokens after a fixed vesting period.
```

Reply:
Indeed, let's prevent using the word locking

Reply:
1. Fair point, as many discord users have asked about locking period too
2. Ok with adding DAI bond to the mix. But re-reading the sentence, do you find the repeated use of "token" sound monotonous? Thinking of removing it in some places (e.g. stake $OHM tokens for more $OHM), but may cause inconsistency

Reply:
Feel free to use the `token` wording everywhere. Using the cash tags should be good enough IMO.